### ISSUE TITLE
Choose older edgedb versions on init

### ISSUE THREAD

**Issue by clmno** [edgedb/edgedb-cli#473]
- EdgeDB CLI Version: EdgeDB CLI 1.0.0-beta.3
- OS Version: Ubuntu

Steps to Reproduce:

1. Delete existing edgedb project
```sh
edgedb project unlink
```
2. Destroy the instance
```sh
edgedb instance destroy edgedb_quickstart
```
3. Init a new project
```sh
edgedb project init
```

Usually after 
```
How would you like to run EdgeDB for this project?
1. Local (native package)
2. Local (docker)
Type a number to select an option:
> 2
```

I get to choose the version of the edgedb server. Which I couldn't here.

*Edit*
We can pass `--version` to select the EdgeDB version
```
edgedb instance create ... --version=1-beta3
```

But this fails silently if the passed in version's docker images in unavailable

**Comment by tailhook** [comment]
Have you been initializing existing project? It get's version from `edgedb.toml` so everybody on the team has the same version. You can then use `project upgrade --to-version` afterwards, or edit toml before initializing if you want something special. But providing users the choice here would only confuse users and will make tutorials more complicated.

**Comment by clmno** [comment]
@tailhook I tried it out again, just to make sure.

No, i don't have an `edgedb.toml` in the directory. But I still failed to choose the EdgeDB version using `edgedb instance create`

```sh
# 1. create a new empty directory
mkdir temp && cd temp
ls -al
total 8
drwxrwxr-x 2 clmno clmno 4096 Aug 16 21:33 ./
drwxrwxr-x 8 clmno clmno 4096 Aug 16 21:33 ../

# 2. Create a new EdgeDB instance
edgedb instance create uno --method=docker
┌──────────────────────┬───────────────────────────────────────────────────┐
│ Instance Name        │ uno                                               │
│ Mode                 │ Docker                                            │
│ Data Directory       │ <docker volume "edgedb_uno">                      │
│ Credentials Path     │ /home/clmno/.config/edgedb/credentials/uno.json │
│ Database Server Port │ 10704                                             │
│ Default User         │ edgedb                                            │
│ Default Database     │ edgedb                                            │
│ EdgeDB Version       │ 1-beta3-451baf4                                   │
└──────────────────────┴───────────────────────────────────────────────────┘

# 3. Create another instance, but this time by choosing --version 
edgedb instance create dos --method=docker --version=1-beta2
┌──────────────────────┬───────────────────────────────────────────────────┐
│ Instance Name        │ dos                                               │
│ Mode                 │ Docker                                            │
│ Data Directory       │ <docker volume "edgedb_dos">                      │
│ Credentials Path     │ /home/clmno/.config/edgedb/credentials/dos.json │
│ Database Server Port │ 10705                                             │
│ Default User         │ edgedb                                            │
│ Default Database     │ edgedb                                            │
│ EdgeDB Version       │ 1-beta3-451baf4                                   │
└──────────────────────┴───────────────────────────────────────────────────┘

```

**Comment by tailhook** [comment]
I've also rechecked. There are only two conditions when we don't ask:
1. Existing instance
2. Non-interactive mode

Maybe you have `edgedb.toml` in one of the parent directories? (we search the parent directories similarly to how `git` does for example).

**Comment by clmno** [comment]
I have a `edgedb.toml` at a child of a parent
```
parent
    |--- temp (is empty)
    |
    |__ quickstart (has edgedb.toml)
```

Ok. Closing this as I'm sure you would have made the due diligence here. Was simply confused. 

ty.

**Comment by clmno** [comment]
I'm still facing issues here. I may be confused. 

I'm trying to run both `1-beta2` and `1-beta3`. But even though I pass `--version` it only picks `1-beta3`
I've even tried creating an instance from `$HOME` but still it defaults to `1-beta3`

**Comment by tailhook** [comment]
> I'm still facing issues here. I may be confused.
> 
> I'm trying to run both `1-beta2` and `1-beta3`. But even though I pass `--version` it only picks `1-beta3`
> I've even tried creating an instance from `$HOME` but still it defaults to `1-beta3`

Do you see something like this: ```Found `edgedb.toml` in `/home/user1/project1` ``` ?

By "it only picks `1-beta3`" you mean that version is prefilled in the input, right? Does it work fine if you edit the version manually?

**Comment by clmno** [comment]
> Do you see something like this: Found `edgedb.toml` in `/home/user1/project1` ?

No I don't

> you mean that version is prefilled in the input, right?

I mean that's the version of the EdgeDB server that gets created (using docker)

> Does it work fine if you edit the version manually?

I did set the `--version` manually. Please refer to the below code block.

Here is the cli command I used + the log
```
edgedb instance create test --method=docker --version=1-beta2
┌──────────────────────┬────────────────────────────────────────────────────┐
│ Instance Name        │ test                                               │
│ Mode                 │ Docker                                             │
│ Data Directory       │ <docker volume "edgedb_test">                      │
│ Credentials Path     │ /home/clmno/.config/edgedb/credentials/test.json   │
│ Database Server Port │ 10706                                              │
│ Default User         │ edgedb                                             │
│ Default Database     │ edgedb                                             │
│ EdgeDB Version       │ 1-beta3-451baf4                                    │
└──────────────────────┴────────────────────────────────────────────────────┘
Initializing EdgeDB instance...
Bootstrapping EdgeDB instance on the local volume...
INFO 18 2021-08-16 18:44:51,063 edb.server: EdgeDB server (version: 1.0-beta.3+g8ab4a7be0.cv202108020000) is starting.
INFO 18 2021-08-16 18:44:52,325 edb.server: Using 78 max backend connections based on total memory.
INFO 18 2021-08-16 18:44:52,345 edb.server.pgcluster: Initializing database cluster in /var/lib/edgedb/data/test
2021-08-16 18:44:53.154 UTC [47] LOG:  starting PostgreSQL 12.7 on x86_64-pc-linux-gnu, compiled by gcc (Debian 8.3.0-6) 8.3.0, 64-bit
2021-08-16 18:44:53.156 UTC [47] LOG:  listening on Unix socket "/var/lib/edgedb/data/test/.s.PGSQL.5432"
2021-08-16 18:44:53.187 UTC [48] LOG:  database system was shut down at 2021-08-16 18:44:52 UTC
2021-08-16 18:44:53.188 UTC [49] FATAL:  the database system is starting up
2021-08-16 18:44:53.191 UTC [47] LOG:  database system is ready to accept connections
INFO 18 2021-08-16 18:44:54,230 edb.server: Creating template database...
INFO 18 2021-08-16 18:44:54,746 edb.server: Creating the necessary PostgreSQL extensions...
INFO 18 2021-08-16 18:44:54,749 edb.server: Initializing the standard library...
INFO 18 2021-08-16 18:44:59,346 edb.server: Creating database: __edgedbsys__
INFO 18 2021-08-16 18:44:59,673 edb.server: Creating database: edgedb
INFO 18 2021-08-16 18:44:59,894 edb.server: Restarting server to reload configuration...
2021-08-16 18:44:59.897 UTC [47] LOG:  received fast shutdown request
2021-08-16 18:44:59.899 UTC [47] LOG:  aborting any active transactions
2021-08-16 18:44:59.901 UTC [47] LOG:  background worker "logical replication launcher" (PID 55) exited with exit code 1
2021-08-16 18:44:59.902 UTC [50] LOG:  shutting down
2021-08-16 18:44:59.928 UTC [47] LOG:  database system is shut down
2021-08-16 18:45:00.016 UTC [64] LOG:  starting PostgreSQL 12.7 on x86_64-pc-linux-gnu, compiled by gcc (Debian 8.3.0-6) 8.3.0, 64-bit
2021-08-16 18:45:00.017 UTC [64] LOG:  listening on Unix socket "/var/lib/edgedb/data/test/.s.PGSQL.5432"
2021-08-16 18:45:00.043 UTC [65] LOG:  database system was shut down at 2021-08-16 18:44:59 UTC
2021-08-16 18:45:00.047 UTC [64] LOG:  database system is ready to accept connections
INFO 18 2021-08-16 18:45:00,678 edb.server: Connection established to backend database: edgedb
INFO 18 2021-08-16 18:45:00,981 edb.server: Generating self-signed TLS certificate.
INFO 18 2021-08-16 18:45:04,177 edb.server: started 4 compiler worker processes
INFO 18 2021-08-16 18:45:04,204 edb.server: Connection established to backend database: edgedb
INFO 18 2021-08-16 18:45:04,210 edb.server: Serving on 127.0.0.1:41183
INFO 18 2021-08-16 18:45:04,211 edb.server: Serving admin on /tmp/tmp.4u25Kf8Dkp/.s.EDGEDB.admin.41183
INFO 18 2021-08-16 18:45:07,261 edb.server: Received signal: Signals.SIGTERM.
INFO 18 2021-08-16 18:45:07,262 edb.server: Shutting down.
2021-08-16 18:45:07.327 UTC [64] LOG:  received fast shutdown request
2021-08-16 18:45:07.333 UTC [64] LOG:  aborting any active transactions
2021-08-16 18:45:07.334 UTC [64] LOG:  background worker "logical replication launcher" (PID 71) exited with exit code 1
2021-08-16 18:45:07.348 UTC [66] LOG:  shutting down
2021-08-16 18:45:07.400 UTC [64] LOG:  database system is shut down
a3da189bd6a024ad8de54d934eb784c9c0f0caa54983b1cccc61ec5c1d4dcc5f
To connect, run:
  edgedb -I test
```

**Comment by 1st1** [comment]
@tailhook Please take a look at this one and decide if it can be closed or we need to investigate more.

**Comment by clmno** [comment]
@tailhook you may close this. I did recreate it (as shown in the previous comment) but I've moved on and this is no longer an issue.

**Comment by tailhook** [comment]
@clmno, sorry I've somehow missed your previous message and previously was distracted by project commands.

So, yes there is a bug when you use `--version` together with `--method`in `edgedb instance create`. And it's fixed by #482.

*(That said, I couldn't find any issue with `project init` command, it installs correct version for me and prompts as needed)*.
